runtime/development/src/evm.rs

```diff
@@ -61,30 +62,35 @@ impl pallet_evm::Config for crate::Runtime {
 	type PrecompilesType = Development<Self>;
 	type PrecompilesValue = PrecompilesValue;
 	type Runner = pallet_evm::runner::stack::Runner<Self>;
-	type RuntimeEvent = crate::RuntimeEvent;
+	type RuntimeEvent = RuntimeEvent;
 	type WeightPerGas = WeightPerGas;
 	type WithdrawOrigin = EnsureAddressTruncated;
 }
 
-impl pallet_evm_chain_id::Config for crate::Runtime {}
+impl pallet_evm_chain_id::Config for Runtime {}
 
 parameter_types! {
 	pub DefaultBaseFeePerGas: U256 = U256::from(1_000_000_000);
 	pub DefaultElasticity: Permill = Permill::from_parts(125_000);
 }
 
-impl pallet_base_fee::Config for crate::Runtime {
+impl pallet_base_fee::Config for Runtime {
 	type DefaultBaseFeePerGas = DefaultBaseFeePerGas;
 	type DefaultElasticity = DefaultElasticity;
-	type RuntimeEvent = crate::RuntimeEvent;
+	type RuntimeEvent = RuntimeEvent;
 	type Threshold = BaseFeeThreshold;
 }
 
-impl pallet_ethereum::Config for crate::Runtime {
-	type RuntimeEvent = crate::RuntimeEvent;
+impl pallet_ethereum::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
 	type StateRoot = pallet_ethereum::IntermediateStateRoot<Self>;
 }
 
-impl pallet_ethereum_transaction::Config for crate::Runtime {
-	type RuntimeEvent = crate::RuntimeEvent;
+impl pallet_ethereum_transaction::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+}
+
+impl axelar_gateway_precompile::Config for Runtime {
```

Comment:
Moved this here since it falls into the evm-related-pallets category and I did that right away on Altair, so it's consistent across the two runtimes